source-code/sdk-js/src/helpers/ast.test.ts

```diff
@@ -25,22 +24,22 @@ describe("ast - wrapVariableDeclaration", () => {
             })
         `
 		const inputAst = parse(input, acornOptions) as unknown as Program
-		const resultAst = await wrapVariableDeclaration(inputAst, "load", "wrapFn")
+		const resultAst = wrapVariableDeclaration(inputAst, "load", "wrapFn")[0]
 		const expectedAst = parse(expected, acornOptions) as unknown as Program
 		const expectedCode = generate(expectedAst)
-		const resultCode = generate(resultAst)
+		const resultCode = generate(resultAst!)
 		expect(resultCode).toEqual(expectedCode)
 	})
-	test("Return error for nonexistent declarator", async () => {
+	test("Return error for nonexistent declarator", () => {
 		const input = `
             export const load = () => {
                 return { data: true }
             }
         `
 		// @ivanohofer - what do you think of the type casting below?
 		const inputAst = parse(input, acornOptions) as unknown as Program
-		const result = wrapVariableDeclaration(inputAst, "blue", "wrapFn")
-		await expect(() => result).rejects.toThrowError("Couldn't find variable declarator.")
+		const result = wrapVariableDeclaration(inputAst, "blue", "wrapFn")[1]
```

Comment:
```suggestion
		const [, exception] = wrapVariableDeclaration(inputAst, "blue", "wrapFn")
```

and for all following function calls